### ISSUE TITLE
Web CLI should not display "Internal Server Error" on bundle add with alias

### ISSUE THREAD

**Issue by nikhilxb** [codalab/codalab-worksheets#456]
```
CodaLab> cl add bundle main::word-vectors//glove.840B.300d .
Internal Server Error
CodaLab> cl add bundle word-vectors//glove.840B.300d .
CodaLab> cl alias
You are only allowed to execute command "alias" from the CLI.
CodaLab> cl add bundle worksheets.codalab.org::word-vectors//glove.840B.300d .
Internal Server Error
CodaLab> cl add bundle https://worksheets.codalab.org::word-vectors//glove.840B.300d .
Internal Server Error
```

**Comment by nikhilxb** [comment]
Closed by codalab-cli/f46c9e2aa2a1b9ef357d70b72909dee8e8b6a39b.

**Comment by w0nche0l** [comment]
Issue moved to [codalab/codalab-cli #1042](https://github.com/codalab/codalab-cli/issues/1042) via [**ZenHub**](https://www.zenhub.com/)
